keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -253,38 +268,66 @@ type ExecCredential struct {
 type ExecCredentialSpec struct {
   // Cluster contains information to allow an exec plugin to communicate with the
   // kubernetes cluster being authenticated to.
```

Comment:
nit: note this is only set if `provideClusterInfo: true` in the exec provider config